### DISCUSSION TITLE
Fehler in der GUI

### DISCUSSION THREAD

**Discussion by zagor1501** [community/community#67892] [GitHub Education]
### Hi everyone,

![image](https://github.com/community/community/assets/129525071/76a0df14-fcb9-4dbc-b19f-0f9332edc318)

**Comment by samus-aran** [comment]
Hi @zagor1501 ! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
